tests/syntax.test.js

```diff
@@ -580,7 +597,7 @@ describe('translate', () => {
   test('turn left: de -> en', () => {
     const b = parseBlock('drehe dich nach rechts um (45) Grad', optionsFor('de'))
     b.translate(allLanguages.en)
-    expect(b.stringify()).toEqual('turn cw (45) degrees')
+    expect(b.stringify()).toEqual('turn right (45) degrees')
```

Comment:
According to a wiki comment I wrote 5 years ago (!), `cw` is preferred:

> FWIW, the guideline is "when flag clicked" and "turn cw _ degrees"; those versions should be considered canonical.

Is it possible to preserve that behaviour?

Reply:
It should be as easy as changing orders of items in extra_aliases.js

Reply:
By the way, can you link to that comment?

Reply:
https://en.scratch-wiki.info/wiki/Scratch_Wiki_talk:Block_Plugin_Style_Guide